include/SSVOpenHexagon/Data/StyleData.hpp

```diff
@@ -33,8 +34,12 @@ class StyleData
     float pulseIncrement;
     bool huePingPong;
     float maxSwapTime;
+    int colorPosOffset;
```

Comment:
This needs to be initialized, otherwise reading from it would be undefined behavior.